src/Groups/abelian_aut.jl

```diff
@@ -154,18 +227,47 @@ function _orthogonal_group(T::TorQuadModule, gensOT::Vector{ZZMatrix}; check::Bo
   return aut
 end
 
-function Base.show(io::IO, aut::AutomorphismGroup{TorQuadModule})
-  T = domain(aut)
+function Base.show(io::IO, ::MIME"text/plain", OT::AutomorphismGroup{TorQuadModule})
   io = pretty(io)
-  n = ngens(aut)
-  print(IOContext(io, :compact => true), "Group of isometries of ", Lowercase(), T, " with ", ItemQuantity(n, "generator"))
+  println(io, "Orthogonal group of", Indent())
```

Comment:
I disagree with renaming this to orthogonal group:

The reason is that the orthogonal group is the group of *all* isometries. But this class frequently represents subgroups of the orthogonal groups.  If we want to call them orthogonal group, we need to set a flag at creation, that this is indeed the full orthogonal group. Any maybe call the other ones subgroups or whatever.

Reply:
I understand. Then I can revert back to `Group of isometries`

Reply:
Committed